### PR TITLE
Fw 3599 word/phrase add category modal

### PR DESCRIPTION
**Pull request by Cara-Barter**
#### Definition of Done
- [ ] modal functional and styled

### CODE REVIEW

File: frontend/app_v2/src/components/CategoriesBrowser/CategoriesBrowserPresentation.js [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+CategoriesBrowserPresentation.propTypes = {
+  categories: array,
+  isLoading: bool,
+  site: object,
+  sitename: string,
+  chooseDocHandler: func,
+  currentCategory: object,
+  setCurrentCategory: func,
+  filteredCategories: array,
+  query: string,
```
Comment (gmcauliffe) [inline frontend/app_v2/src/components/CategoriesBrowser/CategoriesBrowserPresentation.js:86]: Please remove the props from this list that are no longer being used/passed.

File: frontend/app_v2/src/components/CategoriesBrowser/CategoriesBrowserData.js [hunk 2]
```diff
@@ -0,0 +1,69 @@
+import { useState } from 'react'
+import { useQuery } from 'react-query'
+import { useParams } from 'react-router-dom'
+
+//FPCC
+import { useSiteStore } from 'context/SiteContext'
+import api from 'services/api'
+
+function CategoriesBrowserData() {
+  const { site } = useSiteStore() //language site data, basic info that we need often, name, logo, uids ...
```
Comment (gmcauliffe) [inline frontend/app_v2/src/components/CategoriesBrowser/CategoriesBrowserData.js:10]: Please remove these comments/notes before merging.

**Review by gmcauliffe — CHANGES_REQUESTED** [review]
Great work on this component 👍 
A couple of small things to clean up and then it will be good to merge.

File: frontend/app_v2/src/components/CategoriesBrowser/CategoriesBrowserData.js (hunk 2 above)
Comment (Cara-Barter) [reply frontend/app_v2/src/components/CategoriesBrowser/CategoriesBrowserData.js]: thank you, cleaned that up

**Review by gmcauliffe — APPROVED** [review]

**Pull request merged**